.chloggen/elasticsearchexporter_otel-mode-traces.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: elasticsearchexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add OTel mapping mode for traces, update OTel mapping mode for logs
```

Comment:
same feedback as on the metrics PR, any chance you can add more to this note to explain what this change is about?

Reply:
OTel mapping mode, aka OTel native mapping mode, is a new mapping mode that is actively under development, where it maps to a model that's very close to the actual OTel data model, unlike ECS mode where a lot of translations are done in elasticsearchexporter. See [elasticsearchexporter docs for details](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/elasticsearchexporter#elasticsearch-document-mapping). We don't have public docs about mapping that we can point to yet, as it is all under active development. It requires Elasticsearch native support to magically keep the existing UIs working.

There's an initial implementation for logs (done in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33290) as you may have noticed, and here comes the traces (this PR) and metrics (https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34248) counterparts.